### DISCUSSION TITLE
Apple ID Login

### DISCUSSION THREAD

**Discussion by disruptivepatternmaterial** [nVuln/homebridge-lg-thinq#141] [Q&A] [answered]
I use login with Apple ID and I can find the email that it generated when it created the account on the ThinQ app but the password is lost on me. Where might that be stored? I then can use it to generate a token.

**Comment by nVuln** [comment, marked as answer]
you can create a sub account for plugin
following this: https://github.com/nVuln/homebridge-lg-thinq/wiki/Plugin-Authorization-Setting

**Reply by disruptivepatternmaterial** [reply]
I'm aware I can do that, but it seems to suggest I dont need to do that on the wiki page. So I would prefer not to do that if it is actually possible to auth with apple.

**Reply by nVuln** [reply]
at this time we can't use apple id for this plugin, create new account or sub account is easier way to do that
